src/edu/csus/ecs/pc2/core/InternalController.java

```diff
@@ -3020,10 +2954,20 @@ public void start(String[] stringArray) {
             startLog(null, "pc2.startup."+System.currentTimeMillis(), null, null);
         }
         
+        // make sure supplied arguments are recognized (allowed)
+        try {
+            // throw away object to validate allowed options only
+            @SuppressWarnings("unused")
+            ParseArguments checkAllowedArgs = new ParseArguments(stringArray, requireArgumentArgs, AppConstants.allOptionStrings);
```

Comment:
It's not really clear what this new constructor call is intended to do; it would be helpful if the comment lines above gave a more detailed explanation of the purpose of making this call.